package.json

```diff
@@ -32,8 +32,9 @@
     "defaultDryRun": "false"
   },
   "peerDependencies": {
-    "react": ">=0.14.0",
-    "react-dom": ">=0.14.0"
+    "react": "0.14.x || 15.x",
+    "react-dom": "0.14.x || 15.x",
```

Comment:
the actual supported range for `prop-types` is `^0.14.9 || >15.3.0` for both react deps here